webapp/packages/plugin-sql-editor/src/SqlResultTabs/OutputLogs/OutputLogsResource.ts

```diff
@@ -58,6 +58,10 @@ export class OutputLogsResource extends CachedDataResource<IOutputLog[]> {
     this.setData(updatedData);
   }
 
+  reset(): void {
+    this.setData([]);
+  }
```

Comment:
Resources have a `clear` method

Reply:
But this action will clear all logs for all context IDs. I don't think that this is expected behaviour (open two SQL editors for different connections with output log)